### PR TITLE
mock rest + sample draft implementation

### PR DESCRIPTION
**Pull request by mike-hmelov**

### CODE REVIEW

**Review by sergeytrasko — CHANGES_REQUESTED** [review]
Why it's cukes-mock-rest? Is it applicable to pure http as well? Also I believe it can be applicable for graphql as well.

I would propose to rename modules to cukes-http-mock and cukes-http-mock-samples accordingly.

**Comment by mike-hmelov** [comment]
module was renamed to proposed variant

File: cukes-samples/cukes-http-mock-sample/pom.xml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+        <groupId>lv.ctco.cukes</groupId>
+        <artifactId>cukes-samples</artifactId>
+        <version>0.0.22-SNAPSHOT</version>
+    </parent>
+
+    <dependencies>
+        <!-- Cukes Dependencies-->
+        <dependency>
+            <groupId>lv.ctco.cukes</groupId>
+            <artifactId>cukes-mock-rest</artifactId>
```
Comment (sergeytrasko) [inline cukes-samples/cukes-http-mock-sample/pom.xml:18]: Is it correct?

File: pom.xml [hunk 2]
```diff
@@ -19,6 +19,7 @@
         <module>cukes-rabbitmq</module>
         <module>cukes-samples</module>
         <module>cukes-documentation-generator</module>
+		<module>cukes-mock-rest</module>
```
Comment (sergeytrasko) [inline pom.xml:22]: Please adjust it as well

File: cukes-samples/pom.xml [hunk 3]
```diff
@@ -11,6 +11,7 @@
         <module>cukes-rest-sample</module>
         <module>cukes-soap-sample</module>
         <module>cukes-plugin-sample</module>
+		<module>cukes-mock-rest-sample</module>
```
Comment (sergeytrasko) [inline cukes-samples/pom.xml:14]: Please adjust it as well

File: cukes-http-mock/pom.xml [hunk 4]
```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <artifactId>cukes-http-mock</artifactId>
+    <name>Cukes RESTful mock extension</name>
```
Comment (sergeytrasko) [inline cukes-http-mock/pom.xml:7]: Adjust name

**Comment by sergeytrasko** [comment]
Rejecting in favour of PR #132

**Pull request closed without merging**